File: app/src/main/java/movies/test/softserve/movies/viewmodel/FullMovieViewModel.java
package movies.test.softserve.movies.viewmodel;

import android.arch.lifecycle.ViewModel;

import movies.test.softserve.movies.entity.FullMovie;
import movies.test.softserve.movies.entity.Movie;

/**
 * Created by rkrit on 27.10.17.
 */

public class FullMovieViewModel extends ViewModel {

    private Movie movie;

    private FullMovie fullMovie;

    public FullMovie getFullMovie() {
        return fullMovie;
    }

    public void setFullMovie(FullMovie fullMovie) {
        this.fullMovie = fullMovie;
    }

    public Movie getMovie() {
        return movie;
    }

    public void setMovie(Movie movie) {
        this.movie = movie;
    }
}

File: app/src/main/java/movies/test/softserve/movies/viewmodel/FullMovieViewModel.kt
package movies.test.softserve.movies.viewmodel

import android.arch.lifecycle.ViewModel
import movies.test.softserve.movies.entity.Backdrop
import movies.test.softserve.movies.entity.FullMovie
import movies.test.softserve.movies.entity.Poster

/**
 * Created by rkrit on 02.11.17.
 */
class FullMovieViewModel : ViewModel() {
    var fullMovie: FullMovie? = null
    var posters :List<Poster>? = null
    var backdrops : List<Backdrop>? = null
}